cmake/Platform/Common/Install_common.cmake

```diff
@@ -384,15 +384,26 @@ function(ly_setup_cmake_install)
 
     # Collect all Find files that were added with ly_add_external_target_path
     unset(additional_find_files)
+    unset(additional_platform_files)
     get_property(additional_module_paths GLOBAL PROPERTY LY_ADDITIONAL_MODULE_PATH)
     foreach(additional_module_path ${additional_module_paths})
         unset(find_files)
         file(GLOB find_files "${additional_module_path}/Find*.cmake")
         list(APPEND additional_find_files "${find_files}")
+        foreach(find_file ${find_files})
+            # also copy the Platform/<current_platform> to the destination
+            cmake_path(GET find_file PARENT_PATH find_file_parent)
+            unset(plat_files)
+            file(GLOB plat_files "${find_file_parent}/Platform/${PAL_PLATFORM_NAME}/*.cmake")
+            list(APPEND additional_platform_files "${plat_files}")
+        endforeach()
     endforeach()
     install(FILES ${additional_find_files}
         DESTINATION cmake/3rdParty
     )
+    install(FILES ${additional_platform_files}
+        DESTINATION cmake/3rdParty/Platform/${PAL_PLATFORM_NAME}
+    )
```

Comment:
Should we use the `ly_install_files` function for this?

That way the SDK build doesn't try to run this method

Reply:
this method doesnt get call if `LY_INSTALL_ENABLED` is false, the guard is in the root CMakeLists.txt:
```
if(LY_INSTALL_ENABLED)
    # 8. Generate the O3DE find file and setup install locations for scripts, tools, assets etc., required by the engine
    ly_setup_o3de_install()
    # 9. CPack information (to be included after install)
    include(cmake/Packaging.cmake)
endif()
```